cli/js/40_testing.js

```diff
@@ -693,21 +710,8 @@ function test(
   // Delete this prop in case the user passed it. It's used to detect steps.
   delete testDesc.parent;
   const jsError = core.destructureError(new Error());
-  let location;
-
-  for (let i = 0; i < jsError.frames.length; i++) {
-    const filename = jsError.frames[i].fileName;
-    if (filename.startsWith("ext:") || filename.startsWith("node:")) {
-      continue;
-    }
-    location = {
-      fileName: jsError.frames[i].fileName,
-      lineNumber: jsError.frames[i].lineNumber,
-      columnNumber: jsError.frames[i].columnNumber,
-    };
-    break;
-  }
-  testDesc.location = location;
+ 
+  testDesc.location = testDesc[Symbol.for("Deno.test.location")] || getTestCaseLocation();
```

Comment:
I think this might have to use `testDesc.fn[Symbol.for("Deno.test.location")]`.

Also please take a look through this file, there are other instances that look up location - you should the same code for all of them

Reply:
Oh that makes sense, I'll go through and replace each instance!

Reply:
I changed the code to `testDesc.fn[Symbol.for("Deno.test.location")]`. The only other instance I found of looking up a location was the `stepDesc.location` which I changed to: `stepDesc.location =  stepDesc.fn[Symbol.for("Deno.test.location")] || getTestCaseLocation();`